src/documents/blocks/Signature/SignatureEditor.tsx

```
@@ -0,0 +1,136 @@
+      {otherLines.length > 0 && (
+        <div style={{ fontSize: contentFontSize ?? 12, lineHeight: 1.4 }}>
+          {otherLines.map((c, i) => (
+            <div key={`o-${i}`}>{c}</div>
+          ))}
+        </div>
+      )}
+      {socialParts.length > 0 && (
+        <div style={{ marginTop: 10 }}>{socialParts}</div>
+      )}
+      {disclaimerHtml && (
+        <div style={{ fontSize: '10px', opacity: 0.6, marginTop: 8 }} dangerouslySetInnerHTML={{ __html: disclaimerHtml }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)